src/general/OptionsMenu.cs

```diff
@@ -455,4 +467,16 @@ private void OnColourblindSettingSelected(int index)
         Settings.ColourblindSetting = index;
         Settings.ApplyGraphicsSettings();
     }
+
+    private void OnChromaticAberrationToggled(bool toggle)
+    {
+        Settings.ChromaticEnabled = toggle;
+        Settings.ApplyGraphicsSettings();
+    }
+
+    private void OnChromaticAberrationValueChanged(float amount)
+    {
+        Settings.ChromaticAmount = amount;
+        Settings.ApplyGraphicsSettings();
```

Comment:
I think that it is a bit pointless to have this line here (and also the same thing in the above method).
Once the watchable settings are done, just the above line should be made to trigger the change.